### PR TITLE
Updated osg-flock config for OSG 3.6

### PR DESCRIPTION
**Pull request by rynge**

### CODE REVIEW

File: docs/submit/osg-flock.md [hunk 1]
```diff
@@ -71,12 +71,20 @@ Initial Steps
...
+### Register with COManage 
+The adminstrative contact from the the topology entry needs to register with COManage. 
+Instructions can be found [here](https://opensciencegrid.org/technology/policy/comanage-instructions-user/)
+
+Next is to retrive a token so that the new submit host can authenticate with the Open
+Science Pool manager. Please use your COManage registered and approved identity to
+log into the [OSG Token Registration](https://os-registry.opensciencegrid.org/). Once
+logged in, select `Token on Docker`, and find your registered submit node in the list.
+Follow the instructions (you probably have to do the steps on a host with Docker and as
+root), and once you have the token generated, keep that for later steps.
```
Comment (brianhlin) [inline docs/submit/osg-flock.md:87]: We should mention this in the `Before Starting` that they will need access to a host with Docker or Podman

**Review by brianhlin — APPROVED** [review]
One comment, otherwise LGTM

File: docs/submit/osg-flock.md [hunk 2]
```diff
@@ -87,79 +99,96 @@ Start by setting up the OSG YUM repositories following the
...
-1. Under the section `Title2` make sure to set the following (if not already there):
-  
+1. Add a section to map unknown records to OSG:
+        
         :::xml
-        MapUnknownToGroup="1"
-        MapGroupToRole="1"
         VOOverride="OSG"
+        MapGroupToRole="1"
+        MapUnknownToGroup="1"
```
Comment (brianhlin) [inline docs/submit/osg-flock.md:145]: This should be automatically handled by the packaging now: https://github.com/opensciencegrid/osg-flock/blob/master/rpm/osg-flock.spec#L36-L52

Comment (rynge) [reply docs/submit/osg-flock.md]: Ah, fixed.

File: docs/submit/osg-flock.md [hunk 3]
```diff
@@ -67,16 +67,28 @@ contribution and/or research goals.
...
+well as end users of the system. The polices can be found in the [Acceptable Usage Policy document](ap-ospool-aup.md).
+
 ### Register your access point in OSG Topology
 To be part of OSG, your access point should be registered with the OSG.
 You will need information like the hostname, and the administrative and security contacts.
 Follow the [general registration instructions](../common/registration.md#new-resources).
-For historical reasons, the service type is `Submit Node`.
+For historical reasons, the service type is `Submit Node`. We also request that you tag
+the resources with `OSPool`. An example of a registration is
+[the osg-vo.isi.edu entry](https://github.com/opensciencegrid/topology/blob/master/topology/University%20of%20Southern%20California/Information%20Sciences%20Institute/ISI.yaml)
```
Comment (brianhlin) [inline docs/submit/osg-flock.md:80]:
```suggestion
[the osg-vo.isi.edu entry](https://github.com/opensciencegrid/topology/blob/7a71dd4731bb5259f5d9d4004b2df1ddb2bd22ce/topology/University%20of%20Southern%20California/Information%20Sciences%20Institute/ISI.yaml#L32-L57)
```

Permalink for posterity

**Review by brianhlin — APPROVED** [review]
One suggestion but otherwise this LGTM

**Pull request merged**